Corrections.py

```diff
@@ -1,13 +1,17 @@
 import os
 import ROOT
 import yaml
+import itertools
 from RunKit.sh_tools import sh_call
+from Common.Utilities import *
 
 from .tau import TauCorrProducer
 from .met import METCorrProducer
 from .pu import puWeightProducer
 from .CorrectionsCore import *
 
+for wpcl in [WorkingPointsTauVSe,WorkingPointsTauVSmu,WorkingPointsTauVSjet]:
+    ROOT.gInterpreter.Declare(f'{generate_enum_class(wpcl)}')
```

Comment:
this code should be in  baseline Initialize, no in corrections

Reply:
ok